loadshed/cpu.go

```diff
@@ -0,0 +1,52 @@
+package loadshed
+
+import (
+	"context"
+	"math/rand"
+	"time"
+
+	"github.com/gofiber/fiber/v2/internal/gopsutil/cpu"
```

Comment:
Here too. Add gopsutil as an external package

Reply:
Can you give me a bit details on what you mean by "adding it as an external package".

Reply:
> Can you give me a bit details on what you mean by "adding it as an external package".

Use [this package](https://github.com/shirou/gopsutil) instead

Reply:
I see... Thanks!